en/assembly/quick_start_pixhawk4.md

```diff
@@ -26,33 +30,29 @@ Pixhawk 4 should be mounted on the frame using vibration-damping foam pads (incl
+> **Note** The GPS module's integrated safety switch is enabled *by default* (when enabled, PX4 will not let you arm the vehicle). To disable the safety press and hold the safety switch for 1 second. You can press the safety switch again to enable safety and disarm the vehicle (this can be useful if, for whatever reason, you are unable to disarm the vehicle from your remote control or ground station).
 
 
 
 ## Power
 
-Connect the output of the *Power Management Board* (PM board) that comes with the kit to one of the **POWER** bricks of Pixhawk 4 using a 6-wire cable. The PM input **2~12S** will be connected to your LiPo battery. The connections of Power Management Board, including power supply and signal connections to the ESCs and servos, are explained in the table below. Note that the PM board does not supply power to the servos via + and - pins of **FMU PWM-OUT**. 
+Connect the output of the *Power Management Board* (PM board) that comes with the kit to one of the **POWER** bricks of *Pixhawk 4* using a 6-wire cable. The PM input **2~12S** will be connected to your LiPo battery. The connections of Power Management Board, including power supply and signal connections to the ESCs and servos, are explained in the table below. Note that the PM board does not supply power to the servos via + and - pins of **FMU PWM-OUT**.  
 
-The image below shows the power management board provided with Pixhawk 4.
+The image below shows the power management board provided with *Pixhawk 4*.
 
-![Pixhawk 4 - Power Management Board](../../images/pixhawk4_power_management_board.png)
+![Pixhawk 4 - Power Management Board](../../assets/flight_controller/pixhawk4/pixhawk4_power_management_board.jpg)
 
-> **Note** If using a plane or rover, the 8 pin power (+) rail of **FMU PWM-OUT** will need to be separately powered in order to drive 
-  servos for rudders, elevons etc. To do this, the power rail needs to be connected to a BEC equipped ESC or a standalone 5V BEC or a
-  2S LiPo battery. Be careful with the voltage of servo you are going to use here.
+> **Note** If using a plane or rover, the 8 pin power (+) rail of **FMU PWM-OUT** will need to be separately powered in order to drive servos for rudders, elevons etc. To do this, the power rail needs to be connected to a BEC equipped ESC or a standalone 5V BEC or a 2S LiPo battery. Be careful with the voltage of servo you are going to use here.
 
 PIN&Connector | Function
 --- | ---
-I/O PWM-IN | See note below for connection to Pixhawk 4
+I/O PWM-IN | See note below for connection to *Pixhawk 4*

```

Comment:
For this whole table, I understand why you have shown signal wire connections here. The questions I would ask as a newbie are 
- "why do we connect signal wire to esc", 
- "Why don't we have to power the output rail for MC but we do for FW

Diagram to explain how this PMB fits would be nice. Labels so we can clearly work out where the pins are on board. Some of this info could go down into the Motors section maybe?

Reply:
Also "See note below for connection to" - would be good to link to that note.